projects/ui/dialog/body/dialog-actions.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+export type CkDialogActionAlignment = VariantProps<
+  typeof dialogActionsStyles
+>['align']
+
+export const dialogActionsStyles = cva(
+  'flex flex-col sm:flex-row items-center gap-3 px-6 pb-6 pt-3',
+  {
+    variants: {
+      align: {
+        start: 'justify-start',
+        center: 'justify-center',
+        end: 'justify-end',
+        stretch: '[&_>_*]:w-full'
```

Comment:
There are no any background.

Reply:
Yea, that is by design. Thought it looks nice 
![image](https://github.com/user-attachments/assets/81d8ff9b-4cdc-49ef-8385-1b88a61b52c7)

Reply:
Dialog overall looks "heavy" if both header and footer have backgrounds
![image](https://github.com/user-attachments/assets/5a53f07e-f754-456b-9229-1fa57e19e74a)

Reply:
In the first screen buttons are merge with the text. Possible option is to make scrollable content only.